src/main/java/net/dries007/tfc/client/model/entity/LobsterModel.java

```diff
@@ -23,50 +30,68 @@ public static LayerDefinition createBodyLayer()
 
         PartDefinition clawBottomLeft = clawLeft.addOrReplaceChild("clawBottomLeft", CubeListBuilder.create().texOffs(14, 2).addBox(0.0F, 0.0F, -2.0F, 1.0F, 1.0F, 2.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(0.0F, 0.0F, 0.0F, 0.2182F, 0.0F, 0.0F));
 
         PartDefinition clawTopLeft = clawLeft.addOrReplaceChild("clawTopLeft", CubeListBuilder.create().texOffs(14, 14).addBox(0.0F, -1.0F, -2.0F, 1.0F, 1.0F, 2.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(0.0F, 0.0F, 0.0F, -0.2618F, 0.0F, 0.0F));
 
-        PartDefinition armRight = body.addOrReplaceChild("armRight", CubeListBuilder.create().texOffs(0, 9).addBox(-1.0F, 0.0F, -3.0F, 1.0F, 1.0F, 3.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(-2.0F, -2.0F, -3.0F, 0.0F, 0.5236F, 0.0F));
+        PartDefinition armRight = body.addOrReplaceChild("armRight", CubeListBuilder.create().texOffs(0, 9).addBox(-1.0F, 0.0F, -3.0F, 1.0F, 1.0F, 3.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(-1.5F, -2.0F, -3.0F, 0.0F, 0.5236F, 0.0F));
 
         PartDefinition clawRight = armRight.addOrReplaceChild("clawRight", CubeListBuilder.create(), PartPose.offsetAndRotation(0.0F, 0.0F, -3.0F, 0.0F, -0.3054F, 0.0F));
 
         PartDefinition clawBottomRight = clawRight.addOrReplaceChild("clawBottomRight", CubeListBuilder.create().texOffs(6, 12).addBox(-1.0F, 0.0F, -2.0F, 1.0F, 1.0F, 2.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(0.0F, 0.0F, 0.0F, 0.2182F, 0.0F, 0.0F));
 
         PartDefinition clawTopRight = clawRight.addOrReplaceChild("clawTopRight", CubeListBuilder.create().texOffs(10, 13).addBox(-1.0F, -1.0F, -2.0F, 1.0F, 1.0F, 2.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(0.0F, 0.0F, 0.0F, -0.2618F, 0.0F, 0.0F));
 
-        PartDefinition legsLeft = body.addOrReplaceChild("legsLeft", CubeListBuilder.create(), PartPose.offset(1.0F, -1.0F, -1.0F));
+        PartDefinition legsLeft = body.addOrReplaceChild("legsLeft", CubeListBuilder.create(), PartPose.offset(1.5F, -1.0F, -1.0F));
 
         PartDefinition cube_r1 = legsLeft.addOrReplaceChild("cube_r1", CubeListBuilder.create().texOffs(4, 7).addBox(0.0F, 0.0F, -2.5F, 2.0F, 0.0F, 5.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(0.0F, 0.0F, 0.0F, 0.0F, 0.0F, 0.6109F));
 
-        PartDefinition legsRight = body.addOrReplaceChild("legsRight", CubeListBuilder.create(), PartPose.offset(-2.0F, -1.0F, -1.0F));
+        PartDefinition legsRight = body.addOrReplaceChild("legsRight", CubeListBuilder.create(), PartPose.offset(-1.5F, -1.0F, -1.0F));
 
         PartDefinition cube_r2 = legsRight.addOrReplaceChild("cube_r2", CubeListBuilder.create().texOffs(0, 7).addBox(-2.0F, 0.0F, -2.5F, 2.0F, 0.0F, 5.0F, new CubeDeformation(0.0F)), PartPose.offsetAndRotation(0.0F, 0.0F, 0.0F, 0.0F, 0.0F, -0.6109F));
 
         return LayerDefinition.create(meshdefinition, 32, 32);
     }
+
+    //Animation transubstantiated by Troggacompany
```

Comment:
lets not put credits like this inside the code, that's what commit histories are for